apps/uriparser.cpp

```diff
@@ -238,10 +242,10 @@ int main( int argc, char** argv )
     cout << "PORT: " << parser.portno() << endl;
     cout << "PATH: " << parser.path() << endl;
     cout << "PARAMETERS:\n";
-    for (auto p: parser.parameters())
+    for (auto& p: parser.parameters()) {
```

Comment:
Please keep the current brace placement convention.

Reply:
Fixed.